src/consensus/aft/raft.h

```diff
@@ -1853,6 +1939,7 @@ namespace aft
 
       // After entries have been deserialised, try to commit the leader's
       // commit index and update our term history accordingly
+      LOG_DEBUG_FMT("Leader's commit index: {}", r.leader_commit_idx);
```

Comment:
Do we need to keep this log line?

Reply:
No, and there are lots of other logs that should be removed too.